### DISCUSSION TITLE
operator-modulo

### DISCUSSION THREAD

**Discussion by micttyl** [carbon-language/carbon-lang#2129] [Explorer]
Shouldn't [operator-modulo.carbon](https://github.com/carbon-language/carbon-lang/blob/trunk/explorer/testdata/operator/operator-modulo.carbon) be in [operators](https://github.com/carbon-language/carbon-lang/tree/trunk/explorer/testdata/operators) like all other operator tests?

**Comment by jonmeow** [comment]
Yeah, filed #2130
